### PR TITLE
Milestone3 : Trials

### PR DESCRIPTION
**Pull request by derrick1451**
Landing Page
in this module 
1 We used HTML to structure our page
2 We used CSS for styling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# project-trials
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (derrick1451) [inline README.md:5]: Please add a descriptive Readme file to make your work professional

**Review by derrick1451 — COMMENTED** [review]
Hi
Good job so far working on the project requirements! 
there are some issues you still need to work out
Required Changes :recycle: 

1. Please add a descriptive and professional PULL REQUEST.
2. Please add the linter files in your repository and follow the correct GitHub flow
3. please add the gitignore file in your repo 
Cheers and Happy coding :+1: 

-

File: README.md (hunk 1 above)
Comment (muneebulrehman) [reply README.md:5]: Your comment should always sound as a suggestion and not an order, let's keep that in mind. Ok !  😊

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @derrick1451 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

- Always appreciate the person for what he did well before requesting for the changes to be made.
- You can leave a comment or two about the code written, in this case about HTML and CSS like  "How about we delete the commented code because it will clutter our files later". There is still some scope on which you can review the project further.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by derrick1451 — COMMENTED** [review]
Hy there :+1: 
Good job so far :clap: 
there are some issues you still need to work on to go to the next project but you're almost there!
#Highlights
:100: You implemented the header section in a professional way
:100: Good HTML semantic tags
:100: used CSS selectors correctly
#Required Changes :recycle: 
1. Please add a descriptive and professional PULL REQUEST.
2. Please add the linter files in your repository and follow the correct GitHub flow
3. please add the gitignore file in your repo
Cheers and Happy coding +1

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @derrick1451 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

- [ ] As requested by the previous reviewer, your comments could sound less commanding by making them sound like suggestions.
- [ ] The comments should also explain why it is important to make the requested change

E.g
Your comment 
image
Suggessted comment
You could say something like, "Kindly add linter configuration files to the project as this will ensure your code follows and applies industry coding standards".

- [ ] Also at least use an inline comment once as it is part of the requirements. [Follow this link for more info on how to use Github inline comments](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by derrick1451 — COMMENTED** [review]
Hy there :+1: 
Good job so far :clap: 
there are some issues you still need to work on to go to the next project but you're almost there!
# Highlights

-  You implemented the header section in a professional way

-  Good HTML semantic tags

-  used CSS selectors correctly

# Required Changes :recycle: 

- Kindly add linter configuration files to your project as this will ensure your code is not flagged for programming errors and bugs
- Please write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. this helps code reviewers, programmers and recruiters to know the changes you have implemented on the repository

Cheers and Happy coding +1

File: README.md (hunk 1 above)
Comment (danmainah) [reply README.md:5]: - [ ] [OPTIONAL] @derrick1451 great work, I would suggest you update your comment, I can see you created a similar comment to implement the changes requested.

**Review by danmainah — APPROVED** [review]
Hi @derrick1451 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Good design of your review summary
✔️  Good use of inline comments.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**